packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -532,6 +537,26 @@ export class SigningCosmWasmClient extends CosmWasmClient {
     return this.signAndBroadcast(senderAddress, [sendMsg], fee, memo);
   }
 
+  public async cancelUnbondingDelegation(
+    delegatorAddress: string,
+    validatorAddress: string,
+    amount: Coin,
+    creationHeight: bigint | undefined,
+    fee: StdFee | "auto" | number,
+    memo = "",
+  ): Promise<DeliverTxResponse> {
+    const cancelUnbondingDelegationMsg: MsgCancelUnbondingDelegationEncodeObject = {
+      typeUrl: "/cosmos.staking.v1beta1.MsgCancelUnbondingDelegation",
+      value: MsgCancelUnbondingDelegation.fromPartial({
+        delegatorAddress: delegatorAddress,
+        validatorAddress: validatorAddress,
+        amount: amount,
+        creationHeight: creationHeight,
+      }),
+    };
+    return this.signAndBroadcast(delegatorAddress, [cancelUnbondingDelegationMsg], fee, memo);
+  }
```

Comment:
Let's not add more of those helper functions. This is a pretty unsustainable design from long time ago. We have way too many message types these days. A user can just use `signAndBroadcast`.

Reply:
removed